CDTDatastore/CDTDatastore.m

```diff
@@ -4,6 +4,7 @@
 //
 //  Created by Michael Rhodes on 02/07/2013.
 //  Copyright (c) 2013 Cloudant. All rights reserved.
+//  Copyright © 2018 IBM Corporation. All rights reserved.
```

Comment:
Should the latest copyright date be on a separate line or should it be similar to the other libraries e.g. `Copyright (C) 2013, 2018 IBM Corp. All rights reserved.`